boilerplate/flyte/golang_support_tools/go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/flyteorg/boilerplate
 
-go 1.17
+go 1.16
```

Comment:
can we do the boiler plate merge with latest go version first and then rebase instead of doing another PR to update this

Reply:
done